umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-search-results-json-schema.json

```diff
@@ -0,0 +1,286 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "ItemType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Represents a single item found in search results. It contains some metadata about the item found along with the UMM representing the item. umm won't be present if the item represents a tombstone.",
+      "properties": {
+        "meta": {
+          "$ref": "#/definitions/MetaType"
+        },
+        "umm": {
+          "$ref": "umm-var-json-schema.json"
+        },
+        "associations": {
+          "$ref": "#/definitions/AssociationsInItemType"
+        }
+      },
+      "required": ["meta"]
+    },
+    "MetaType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "CMR level metadata about the item found. This represents data not found in the actual metadata.",
```

Comment:
I don't know what this means

Reply:
It is the type for meta in UMM_JSON records.  It holds the meta-metadata for a record, and not the actual contents of the metadata.  The contents includes the time the record was ingested, which provider ingested the record, etc.

I didn't change this file. I just brought it forward. Looking at the code base, this was in the initial release 6 years ago.  This file is only used for running tests.  I would rather that it goes away, but we are using it to make sure that the coded response in production is how we meant the search results to be displayed.

Reply:
I wonder if anyone new to our system will understand this, I think it is confusing.